src/addon/workbench/workbench.js

```diff
@@ -112,37 +112,56 @@ cwc.addon.Workbench.prototype.getAllProjects_ = function() {
 
 
 /**
- * @param {Object} projects
- * @param {string=} filterTag
+ * @param {Array<Object>} projects
+ * @param {number=} filterTagId
  * @return {Object}
  * @private
  */
 cwc.addon.Workbench.prototype.filterProjectsByTag_ = function(
-    projects, filterTag) {
+    projects, filterTagId) {
   return projects.filter((project) => (
-    project['theme_tags'].find((tag) => (
-      (tag.title || '').toLowerCase() === (filterTag || '').toLowerCase())
-    )
+    project['tags'].find((tagId) => tagId === filterTagId)
   ));
 };
 
+
 /**
  * @private
  */
 cwc.addon.Workbench.prototype.showRelevantProjects_ = async function() {
   const allProjects = await this.getAllProjects_();
+  const tags = Object.keys(cwc.addon.WorkbenchMap);
   this.clearCards_();
 
-  Object.keys(cwc.addon.WorkbenchMap).forEach((tag) => {
-    const tabNode = document.querySelector(
-      cwc.addon.WorkbenchMap[tag].appendNode);
-    if (!tabNode) return;
-
-    const mode = cwc.addon.WorkbenchMap[tag].mode;
+  tags.forEach((tagName) => {
+    const tagId = cwc.addon.WorkbenchMap[tagName].wbTagId;
+    const mode = cwc.addon.WorkbenchMap[tagName].mode;
     const userLanguage = this.helper.getUserLanguage();
     const userLanguageName = cwc.utils.I18n.getEnglishName()[userLanguage];
-    let matchingProjects = this.filterProjectsByTag_(allProjects.filter(
-      (project) => project.language == userLanguageName), tag);
+    const matchingProjects = this
+        .filterProjectsByTag_(allProjects, tagId)
+        .filter((project) => project.language == userLanguageName);
+    const tabNode = document.querySelector(
+        cwc.addon.WorkbenchMap[tagName].appendNode);
+    const cardListId = `workbench-${mode}-cards`;
+
+    // If no matches or no matching CwC tab for the tag, render nothing
+    if (!tabNode || !matchingProjects || matchingProjects.length < 1) return;
```

Comment:
Generally '!tabNode' will only happen if we mismatch strings in workbench_map.js and the .soy files, correct? How about a warning to help troubleshoot why projects don't show up.
```suggestion
    if (!tabNode) {
      this.log_.warn('No CwC section',
        cwc.addon.WorkbenchMap[tagName].appendNode,
        ' Possibly a typo in the WorkbenchMap');
      return;
    }
    if (!matchingProjects || matchingProjects.length < 1) return;
```

Reply:
Yeah, that should be the only time that would happen, or if a mode/section is removed from CwC. I can't accept this suggestion since the PR was merged, but feel free to make that change if you have a moment. Otherwise, I can make it soon.